includes/class-wc-stripe.php

```diff
@@ -821,13 +821,7 @@ public function get_main_stripe_gateway() {
 			return $this->stripe_gateway;
 		}
 
-		if ( WC_Stripe_Feature_Flags::is_upe_preview_enabled() && WC_Stripe_Feature_Flags::is_upe_checkout_enabled() ) {
-			$this->stripe_gateway = new WC_Stripe_UPE_Payment_Gateway();
-
-			return $this->stripe_gateway;
-		}
-
-		$this->stripe_gateway = new WC_Gateway_Stripe();
+		$this->stripe_gateway = new WC_Stripe_UPE_Payment_Gateway();
 
 		return $this->stripe_gateway;
```

Comment:
Not blocking, but it might be easier to flip the condition here to simplify the code.
```suggestion
		if ( ! $this->stripe_gateway ) {
			$this->stripe_gateway = new WC_Stripe_UPE_Payment_Gateway();
		}

		return $this->stripe_gateway;
```

Reply:
I agree. Updated it in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4718/commits/a1c55f11a25b305c1373f0244640c71c2ef6af3a